### PR TITLE
Add dev scripts (scale, 2818 backup test) to test repo 

### PR DESCRIPTION
**Pull request by joshimoo**
In the future we can turn longhorn.py and common functionality in to a module
that can easily be imported. The scale test was moved from the longhorn/longhorn repo.

longhorn/longhorn#2818

### CODE REVIEW

**Comment by khushboo-rancher** [comment]
This is great!

Can we add readme file for backup test as well so that anyone using it will know the longhorn url, volume count etc variables should be set before running it.

File: manager/dev/backup-test/backup-test.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import sys
+import logging
+
+import longhorn
+import common
+
+# env setup
+NODE_NAME = "jmoody-2818-post-work1"
```
Comment (khushboo-rancher) [inline manager/dev/backup-test/backup-test.py:8]: Can we use a generalized name here?

File: manager/dev/backup-test/backup-test.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+
+def create_backup(volume, snapshot_name):
+    log = logging.getLogger()
+    backup = volume.snapshotBackup(name=snapshot_name)
+    log.info("volume %s created backup %s" % (volume.name, backup.id))
+    return backup
+
+
+def test_1859_backup_status():
```
Comment (khushboo-rancher) [inline manager/dev/backup-test/backup-test.py:76]: Any specific reason for this naming?

**Comment by innobead** [comment]
@joshimoo could you help answer @khushboo-rancher questions ☝️ ?

**Comment by joshimoo** [comment]
Not sure if this is still needed, I opened that back then so it doesn't get lost and I can link to the scale scipts as a basis for @PhanLe1010 :)

File: manager/dev/backup-test/backup-test.py (hunk 2 above)
Comment (joshimoo) [reply manager/dev/backup-test/backup-test.py:76]: It was supposed to refer to a repro/eval of the solution for a specific issue.
But I think I used the wrong issue number, it should have been the grpc socket issue.
https://github.com/longhorn/longhorn/issues/2818

**Comment by joshimoo** [comment]
For now we can close this, issue withhout merging. 
Since the scale test will be / has been taken over by @PhanLe1010 
And the socket backup issue https://github.com/longhorn/longhorn/issues/2818 is no longer relevant.

**Comment by PhanLe1010** [comment]
Scale test script PR is at https://github.com/longhorn/longhorn-tests/pull/834.

**Pull request closed without merging**